kobo/apps/openrosa/apps/logger/migrations/0040_increase_file_fields_max_length.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 4.2.15 on 2024-12-19 21:31
+
+from django.conf import settings
+from django.db import migrations
+import kobo.apps.openrosa.apps.logger.models.attachment
+import kobo.apps.openrosa.apps.logger.models.xform
+import kpi.deployment_backends.kc_access.storage
+import kpi.fields.file
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('logger', '0039_populate_counters'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
What would happen if there exist values with more than 380 characters in this field?

Reply:
they would be truncated, but default is 100. so we are safe